.github/workflows/pkgdown.yaml

```diff
@@ -21,11 +21,11 @@ jobs:
 
       - name: Install dependencies
         run: |
-          install.packages(c("remotes","devtools", "rmarkdown","tidyverse","DT","here", "pkgdown", "textshaping"))
-          devtools::install_github("epiforecasts/scoringutils")
-          remotes::install_deps(dependencies = NA)
+          install.packages(c("devtools","pkgdown"))
+          devtools::install_github("eepiforecasts/scoringutils@v1.2.2")
```

Comment:
I assumed the `scoringutils` version here should match the changes made in https://github.com/reichlab/covidHubUtils/pull/400